### PR TITLE
Fix #917 - Fix flexbox to truncate alias

### LINKED ISSUE #917: “Forwarded” text and the More details button are displayed outside the alias’s white background
**Build:**
Stage

**Affected platforms:**
Windows 10 64-bit
macOS 10.14

**Browsers tested on:**
Firefox

**Steps to reproduce:**

1. Navigate to https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net and log in with an existing account.
2. Add an alias and observe the “Forwarded” text and the More details button.

**Actual result:** 
“Forwarded” text and the More details button are displayed outside the alias’s white background.

**Expected result:** 
“Forwarded” text and the More details button are displayed on the alias’s white background.

text displayed outside the white background

### PR DESCRIPTION
**Pull request by codemist**
**Preview:**
image

### CODE REVIEW

**Comment by groovecoder** [comment]
Spot-check works and looks good. This is a good fix to make as we start working on domain aliases - people might make really long domains.

File: static/scss/partials/dashboard.scss [hunk 1]
```diff
@@ -194,6 +194,9 @@
 
 .c-alias-name {
     margin-left: $spacing-xs;
+    width:100%;
+    overflow:hidden;
```
Comment (maxxcrawford) [inline static/scss/partials/dashboard.scss:198]: nit: white space

```suggestion
    width: 100%;
    overflow: hidden;
```

File: static/scss/partials/main.scss [hunk 2]
```diff
@@ -676,7 +676,7 @@ table {
   height: 100%; /* expand clickable area of relay address */
   display: block;
   position: relative;
-  max-width: 100%;
+  max-width:100%;
```
Comment (maxxcrawford) [inline static/scss/partials/main.scss:679]: nit: white space (which would mean no change to this line)

**Review by maxxcrawford — CHANGES_REQUESTED** [review]

File: privaterelay/templates/profile.html [hunk 3]
```diff
@@ -99,8 +99,8 @@
             </div>
             <!--end enable/disable toggle -->
 
-            <div class="c-alias-name xflx xflx-col">
-              <div class="relay-email-row card-top-row xflx xflx-row inset">
+            <div class="c-alias-name">
+              <div class="relay-email-row card-top-row inset">
```
Comment (maxxcrawford) [inline privaterelay/templates/profile.html:103]: Thanks for cleaning this up! 🧹

File: static/scss/partials/main.scss (hunk 2 above)
Comment (codemist) [reply static/scss/partials/main.scss]: Updated the white space issues!

**Review by maxxcrawford — APPROVED** [review]
Works great!

**Pull request merged**